python/lsst/ts/phosim/utils/ConvertZernikesToPsfWidth.py

```diff
@@ -0,0 +1,126 @@
+# This file is part of ts_phosim.
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import galsim
+import numpy as np
+
+
+def getPsfGradPerZernike(
+    jmax: int = 22,
```

Comment:
do you want to add `jmin` as a kwarg as well? We usually ignore piston,  x,y, tilt, but would be interesting to see their contribution if calculated in some other situation (eg. auxTel testing)

Reply:
That's a good idea! I will add that as well.

Reply:
Done.